src/main/java/frc/robot/posev2/RobotPose.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.posev2;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+
+
+public class RobotPose {
+    private RobotPosition robotPosition;
+    private VisionData visionData;
+    private Pose2d robotPose2d;
+    private WheelColorValue wheelColor;
+    private TargetLocation targetLocation;
+    
+    public RobotPose(double f, double h, double t, VisionData vData){
```

Comment:
f, h, and t are simply used to construct a robotPosition. I'd prefer to sipmly accept a robot position here.